thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -1,56 +1,70 @@
 import Ember from 'ember';
-import { makeIterable, filterObject, filterPrefix, toBaselineUrn, toCurrentUrn } from 'thirdeye-frontend/helpers/utils';
+import { filterObject, filterPrefix, toBaselineUrn, toCurrentUrn } from 'thirdeye-frontend/helpers/utils';
 import EVENT_TABLE_COLUMNS from 'thirdeye-frontend/mocks/eventTableColumns';
 import config from 'thirdeye-frontend/mocks/filterBarConfig';
+import SHA256 from 'cryptojs/sha256';
 
 const ROOTCAUSE_TAB_DIMENSIONS = "dimensions";
 const ROOTCAUSE_TAB_METRICS = "metrics";
 const ROOTCAUSE_TAB_EVENTS = "events";
 
 export default Ember.Controller.extend({
-
-  /**
-   * QueryParams that needs to update the url
-   */
   queryParams: [
-    'granularity',
-    'filters',
-    'compareMode',
-    'anomalyRangeStart',
-    'anomalyRangeEnd',
-    'analysisRangeStart',
-    'analysisRangeEnd'
+    'metricId',
+    'anomalyId',
+    'shareId'
   ],
-  entitiesService: Ember.inject.service('rootcause-entities-cache'), // service
 
-  timeseriesService: Ember.inject.service('rootcause-timeseries-cache'), // service
+  //
+  // services
+  //
+  entitiesService: Ember.inject.service('rootcause-entities-cache'),
+
+  timeseriesService: Ember.inject.service('rootcause-timeseries-cache'),
+
+  aggregatesService: Ember.inject.service('rootcause-aggregates-cache'),
 
-  aggregatesService: Ember.inject.service('rootcause-aggregates-cache'), // service
+  breakdownsService: Ember.inject.service('rootcause-breakdowns-cache'),
 
-  breakdownsService: Ember.inject.service('rootcause-breakdowns-cache'), // service
+  ajax: Ember.inject.service(),
```

Comment:
non-blocking: fyi, since 2.16, ember supports this way of importing modules instead of everything (import Ember from 'ember') 

### Official doc: 
https://www.emberjs.com/blog/2017/10/11/ember-2-16-released.html#toc_ember-js-modules-api

### Usage in our code: 
- https://github.com/linkedin/pinot/blob/master/thirdeye/thirdeye-frontend/app/pods/rca/details/dimensions/route.js#L4
- https://github.com/linkedin/pinot/blob/master/thirdeye/thirdeye-frontend/app/pods/rca/details/dimensions/route.js#L7